user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+---
+uid: DOM_Remove_Enum_Entry
+---
+
+# How to remove an enum entry from a GenericEnumFieldDescriptor
+
+This tutorial shows you how to soft-delete and permanently delete an enum entry from a GenericEnumFieldDescriptor.
+
+## Overview
+
+- [Step 1: Setup DOM](#step-1-setup-dom)
+- [Step 2: Soft-delete the enum entry](#step-2-soft-delete-the-enum-entry)
+- [Step 3: Remove the value from existing instances](#step-3-remove-the-value-from-existing-instances)
+- [Step 4: Remove the enum entry from the DOM definition](#step-4-remove-the-enum-entry-from-the-dom-definition)
+
+The soft-delete option on an enum entry hides that entry to users. That way users are unable to create more DOM instances using that entry. Later on it can be completely removed.
+If the entry is used by a DOM instance, that value needs to be removed from those instances before removing it from the definition. If that enum entry is not used, it can immediately be removed (see [Step 4: Remove the enum entry from the DOM definition](#step-4-remove-the-enum-entry-from-the-dom-definition)).
+
+## Step 1: Setup DOM
+
+This setup is purely for the tutorial, you can use your own setup and apply the other steps to your setup (see [Step 2: Soft-delete the enum entry](#step-2-soft-delete-the-enum-entry)).
+
+The given code does the following:
+1. Creates a DOM module.
+1. Creates a DOM definition containing a GenericEnumFieldDescriptor.
+1. Creates a DOM instance using that enum entry of that DOM definition.
+
+```C#
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.Apps.Modules;
+using Skyline.DataMiner.Net.Apps.Sections.SectionDefinitions;
+using Skyline.DataMiner.Net.GenericEnums;
+using Skyline.DataMiner.Net.ManagerStore;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using Skyline.DataMiner.Net.Sections;
+
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will create a basic people app with one DomInstance.
+    /// Each domInstance for this sectionDefinition will contain:
+    ///     - first name of the person
+    ///     - last name of the person
+    ///     - phone number of the person
+    ///     - job title of the person
+    ///</summary>
+    public class Script
+    {
+        private const string ModuleName = "people_app";
+        private DomHelper _domHelper;
```

Comment:
Having these two private fields seems unnecessary since we only have one method.